libsql-server/src/hrana/ws/session.rs

```diff
@@ -77,16 +76,11 @@ pub(super) async fn handle_initial_hello(
         .with(namespace.clone(), |ns| ns.jwt_key())
         .await??;
 
-    // Convert jwt token into a HeaderValue to be compatible with UserAuthStrategy
-    let user_credential = jwt
-        .clone()
-        .and_then(|t| HeaderValue::from_str(&format!("Bearer {t}")).ok());
-
     let auth = namespace_jwt_key
         .map(Jwt::new)
         .map(Auth::new)
         .unwrap_or(server.user_auth_strategy.clone())
-        .authenticate(UserAuthContext { user_credential })
+        .authenticate(UserAuthContext { scheme: Some("Bearer".into()), token: jwt })
```

Comment:
Related to my comment above this would read better if we had a constructor like `UserAuthContext::from_scheme("Bearer", jwt)`.